dts/arm/st/f7/stm32f7.dtsi

```diff
@@ -324,6 +324,18 @@
 			label = "SPI_6";
 		};
 
+		can1: can@40006400 {
+			compatible = "st,stm32-can";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40006400 0x400>;
+			interrupts = <19 0>, <20 0>, <21 0>, <22 0>;
+			interrupt-names = "TX", "RX0", "RX1", "SCE";
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x02000000>; //RCC_APB1ENR1_CAN1EN
```

Comment:
Not sure a bout the C++ comment style here.

Reply:
Removed it.